tests/conftest.py

```diff
@@ -31,6 +108,36 @@ def job(job_directory):
     return init_job(job_directory)
 
 
+@pytest.fixture
+@freeze_time("2024-01-01T12:00:00")
+def run_directory(job, job_directory):
+    return init_run(job_directory)
+
+
+@pytest.fixture
+def input_file():
+    return "s3://timdex-extract-dev/source/source-2024-01-01-full-extracted-records-to-index.xml"
+
+
+@pytest.fixture
+def output_filename():
+    return "source-2024-01-01-full-transformed-records-to-index.json"
+
+
+@pytest.fixture
+def transformed_directories(run_directory):
+    transformed_directory_a = str(Path(run_directory) / "transformed/a")
+    transformed_directory_b = str(Path(run_directory) / "transformed/b")
+    return transformed_directory_a, transformed_directory_b
+
+
+@pytest.fixture
+def create_transformed_directories(run_directory):
+    return create_directories(
+        base_directory=run_directory, subdirectories=["transformed/a", "transformed/b"]
+    )
```

Comment:
It's worth nothing that these two fixtures are different. 

* `transformed_directories`:  only returns a tuple of strings representing transformed directories A and B.
   * This is used by mocks used by the test for `run_ab_transforms` because the `run_ab_transforms` will create the directories.
   
* `create_transformed_directories`: returns a tuple of strings representing transformed directories A and B _**and**_ actually creates the transformed directories. 
   * This is used by tests for `run_docker_container` and `get_transformed_files`.